app/components/user-cards.js

```diff
@@ -2,7 +2,24 @@ import Component from '@ember/component';
 import { computed } from '@ember/object';
 
 export default Component.extend({
+
+    select_team:"All Employees",
+
     teamsl : computed ('members.@each.team',function(){
         return this.get('members').filterBy('team')
-    }) 
-});
\ No newline at end of file
+    }) ,
+
+    teams: computed ('members.@each.team',function(){
+        return this.members.mapBy('team').uniq()
+    }),
+
+    actions: {
+        
+        getTeams(teamName) {
+            this.set("select_team",teamName);
+            return this.set('teamsl',this.get('members').filterBy('team',teamName) )
```

Comment:
We don't set a computed Property explicitly. We add appropriate triggers that, the values are updated automatically when the trigger values are changed.

Reply:
updated.